h/api/models/test/document_test.py

```diff
@@ -7,15 +9,66 @@
 from h.api.models.document import merge_documents
 
 
+def test_document_title():
+    doc = Document()
+    meta_title = DocumentMeta(type='title', value='The Title', document=doc, claimant='http://example.com')
```

Comment:
This is absolutely fine, I just want to note FYI that you can also do stuff like

```python
doc.meta.append(DocumentMeta(type='title', value='The Title', claimant='http://example.com'))
```

if you prefer.